frontend/src/metabase/public/components/EmbedModal/SelectEmbedTypePane/SelectEmbedTypePane.tsx

```diff
@@ -112,27 +122,46 @@ export function SelectEmbedTypePane({
         </SharingPaneButton>
 
         {/* INTERACTIVE EMBEDDING */}
-        <Link
+        <MaybeLink
           to={interactiveEmbeddingCta.url}
           target={interactiveEmbeddingCta.target}
           rel="noreferrer"
+          shouldRender={
```

Comment:
this new flag controls whether or not we will render an anchor tag

Reply:
I think the name is a bit misleading, I thought it means whether we show or hide this link (akin to `show && <MaybeLink />`). Perhaps `shouldRenderAsLink` or something more explicit might help.

Reply:
I think `shouldRenderLink` also does the trick.

Reply:
Fixed in 7284ae014ec55e8568eb6ced593bdef365da8649